.circleci/config.yml

```diff
@@ -23,7 +30,8 @@ workflows:
   version: 2
   tests:
     jobs:
-      - automated-tests
+      - check-code-style
+      - run-automated-tests
```

Comment:
The CI won't create a container for each job, right? (2 containers for our case).

Reply:
Actually it will create two containers and run the jobs in parallel.